.github/ISSUE_TEMPLATE.md

```diff
@@ -11,13 +11,31 @@ Please fill in as much of the template below as you're able.
 Version: output of `node -v`
 Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
 Subsystem: if known, please specify affected core module name
-
-If possible, please provide code that demonstrates the problem, keeping it as
-simple and free of external dependencies as you are able.
 -->
 
 * **Version**:
 * **Platform**:
 * **Subsystem**:
 
-<!-- Enter your issue details below this comment. -->
+### What steps will reproduce the bug?
+
+<!--
+Enter details about your bug, preferably a simple code snippet that can be
+run using `node` directly without installing third-party dependencies.
+-->
+
+### How often does it reproduce? Is there a required condition?
+
+### What is the expected behavior?
+
+<!--
+If possible please provide textual output instead of screenshots.
+-->
+
+### What do you see instead?
+
+<!--
+If possible please provide textual output instead of screenshots.
+-->
+
+### Additional information
```

Comment:
Should we add a comment under here just to make it clear and to be consistent with the other entries?

```suggestion
### Additional information

<!--
Tell us anything else you think we should know.
-->
```